### DISCUSSION TITLE
Shadow banned from Github Copilot?

### DISCUSSION THREAD

**Discussion by helomri** [community/community#17904] [Copilot Conversations]
Hello,
Today I tried to install the copilot on my Ubuntu 22.04 computer and after some version issues I got this message in the vscode console;
```
[2022-06-04 10:30:11.948] [renderer1] [error] [Extension Host] [ERROR] [default] [2022-06-04T08:30:11.947Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```
If I try to join the waitlist of the github copilot It says that I already have github copilot activated. I hope this is only a bug and not that github shadow ban some users randomly.

**Comment by davecheney** [comment]
I can confirm you invite is active. Please

1.  <img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
2. Double check that you are using VSCode. Linux distros often ship with OSSCode or VSCodium which do _not_ work with Copilot.
3. Upgrade to the latest extension.
